spring-integration-amqp/src/main/java/org/springframework/integration/amqp/outbound/AsyncAmqpOutboundGateway.java

```diff
@@ -41,6 +44,8 @@
 
 	private final MessageConverter messageConverter;
 
+	private volatile boolean requiresReply; // overrides super
```

Comment:
Let me know why we need it here if we can leverage the same in the `AbstractReplyProducingMessageHandler` and `requiresReply` and `asyncReplySupported` combination.
Yes, I won't mind against `protected getRequiresReply()`, but I think we still can rely on the super property, just for cases to avoid confuses during debug.
WDYT?